core/arch/arm/mm/tee_pager.c

```diff
@@ -566,8 +566,8 @@ static void *pager_add_alias_page(paddr_t pa)
 	struct core_mmu_table_info *ti;
 	/* Alias pages mapped without write permission: runtime will care */
 	uint32_t attr = TEE_MATTR_VALID_BLOCK |
-			(TEE_MATTR_CACHE_CACHED << TEE_MATTR_CACHE_SHIFT) |
-			TEE_MATTR_SECURE | TEE_MATTR_PR;
+			(TEE_MATTR_MEM_TYPE_CACHED << TEE_MATTR_MEM_TYPE_SHIFT)
+			| TEE_MATTR_SECURE | TEE_MATTR_PR;
```

Comment:
for consistency in optee_os sources, `|` should be at the end of the previous line.
suggestion:
```c
	uint32_t attr = TEE_MATTR_VALID_BLOCK | TEE_MATTR_SECURE |
			TEE_MATTR_PR | (TEE_MATTR_MEM_TYPE_CACHED <<
					TEE_MATTR_MEM_TYPE_SHIFT);